package.json

```diff
@@ -97,7 +97,7 @@
       "^.+\\.svg$": "<rootDir>/scripts/stubs/fileStub.js",
       "react-transition-group/CSSTransition": "<rootDir>/scripts/stubs/cssTransitionStub.js"
     },
-    "setupFiles": [
+    "setupFilesAfterEnv": [
```

Comment:
This was required to work around a problem with integrating `jest-axe` into our setup: https://github.com/testing-library/jest-dom/issues/122#issuecomment-650520461